lgc/patch/PatchPeepholeOpt.cpp

```diff
@@ -569,94 +569,6 @@ void PatchPeepholeOpt::visitPHINode(PHINode &phiNode) {
 
     return;
   }
-
```

Comment:
I think it is better the deletion goes to a separate patch, that would be easy for bisect or revert if we find something wrong.

Reply:
Done, split the change into two commits.